pkg/rancher-desktop/components/Version.vue

```diff
@@ -0,0 +1,59 @@
+<script lang="ts">
+import Vue from 'vue';
+
+import { ipcRenderer } from '@pkg/utils/ipcRenderer';
+export default Vue.extend({
+  props: {
+    icon: {
+      type:    String,
+      default: '',
+    },
+    isStatusBarItem: {
+      type:    Boolean,
+      default: false,
+    },
+  },
+  data() {
+    return { version: this.t('product.versionChecking') };
+  },
+  computed: {
+    getTooltip(): { content: string, placement: string, classes: string } {
+      return {
+        content:   `<b>${ this.t('product.version') }</b>: ${ this.version }`,
+        placement: 'top',
+        classes:   'tooltip-footer',
+      };
+    },
+  },
+  mounted() {
+    ipcRenderer.on('get-app-version', (event, version) => {
+      this.$data.version = version;
```

Comment:
We shouldn't require the `$data` instance property here and the shorthand `this.version` should suffice. 

```suggestion
      this.version = version;
```